plugins/aglight/colors.js

```diff
@@ -0,0 +1,605 @@
+/** Taken from https://github.com/timoxley/colornames */
```

Comment:
We already use the [color-names](https://github.com/jonathantneal/color-names) npm module in the e:cue plugin: https://github.com/OpenLightingProject/open-fixture-library/blob/schema-12.2.1/plugins/ecue/import.js#L1-L10

Is there a way to use that module here, too?
Or maybe both plugins can be refactored to use another npm module? I'd rather not have a list of color names to maintain myself.

Reply:
I found that module, but I didn't use it as it is 5+ years old... Anyway, the color names haven't changed much 😉
I'll change that 👍

Reply:
What about using this?
[color-name-list](https://github.com/meodai/color-names/) (or on [npm](https://www.npmjs.com/package/color-name-list))

It is maintained and has a lot more colors.

I'm missing the following colors in the color-names npm module: `UV`, `Cold white` and `Warm White`. Of course, I could just add them in the code of the export plugin, but I don't feel like this would be good practise...

Reply:
That module looks good! :)
Would you mind changing the e:cue plugin to that module in a separate PR, too?

Reply:
No problem, I can do that. Thanks for re-deploying